compiler/next/include/chpl/resolution/resolution-types.h

```diff
@@ -242,32 +288,53 @@ struct PoiInfo {
   // This is a set of pairs of (Call ID, Function ID).
   // This includes POI calls from functions called in this Function,
   // transitively
-  std::set<std::pair<ID, ID>> poiFnIdsUsed;
+  std::set<std::pair<ID, ID>> poiFnIdsUsed_;
 
+ public:
   // default construct a PoiInfo
   PoiInfo() { }
 
   // construct a PoiInfo for a not-yet-resolved instantiation
   PoiInfo(const PoiScope* poiScope)
-    : resolved(false), poiScope(poiScope) {
+    : resolved_(false), poiScope_(poiScope) {
   }
   // construct a PoiInfo for a resolved instantiation
   PoiInfo(std::set<std::pair<ID, ID>> poiFnIdsUsed)
-    : resolved(true), poiFnIdsUsed(std::move(poiFnIdsUsed)) {
+    : resolved_(true), poiFnIdsUsed_(std::move(poiFnIdsUsed)) {
+  }
+
+  /** return the poiScope */
+  const PoiScope *poiScope() const { return poiScope_; }
+
+  /** set the poiScope */
+  void setPoiScope(const PoiScope *poiScope) { poiScope_ = poiScope; }
+
+  /** set resolved */
+  void setResolved(bool resolved) { resolved_ = resolved; }
+
+  // TODO callers copy and store this elsewhere, do we return as is? change the
+  // getter to poiFnIdsUsedAsSet? make callers do std::set(poiFnIdsUsed.begin(),
+  // poiFnidsUsed.end()) ?
```

Comment:
I think this is OK for now but `poiFnIdsUsedAsSet` makes sense to me as an improvement

Reply:
Or we could have it create a `std::set` from it, and then gradually change the call sites to something else, if that's not what we wanted (for performance, say).